### PR TITLE
621 add uberspace infrastructure

### PR DESCRIPTION
**Pull request by bascht**
Add service + htaccess + public symlinks for our app and a new `deploy:setup` task that can be run to override / create this whole lot.

### CODE REVIEW

File: config/deploy/master.rb [hunk 1]
```diff
@@ -1 +1,2 @@
 set :branch, 'master'
+set :vhost, 'master.hacken.in'
```
Comment (moonglum) [inline config/deploy/master.rb]: Very cool :+1: That makes a lot of sense :+1:

Comment (moonglum) [inline config/deploy/master.rb]: Should we maybe do the same for the port? But we can't read it from the .htaccesss ,right?

**Comment by moonglum** [comment]
Looks very good. I would love to put the information about the port into one central place, because, you know:

> EVERY PIECE OF KNOWLEDGE MUST HAVE A SINGLE, UNAMBIGU- OUS, AUTHORITATIVE REPRESENTATION WITHIN A SYSTEM.

File: config/deploy/master.rb (hunk 1 above)
Comment (bitboxer) [inline config/deploy/master.rb]: But can't you generate the htaccess on deploy and inject the port in there?

Comment (moonglum) [inline config/deploy/master.rb]: Yep, that's what I thought about as well. Something like an ERB Template.

**Comment by moonglum** [comment]
Can we maybe also link the maintanance page from #602 to the right place?

**Comment by bascht** [comment]
@moonglum @bitboxer glad you fell for my port decoy and insist on EVERY PIECE OF KNOWLEDGE HAVING A SINGLE, UNAMBIGUOUS, AUTHORITATIVE REPRESENTATION WITHIN A SYSTEM.

Can we now please [talk](https://github.com/hacken-in/website/blob/master/.ruby-version#L1) [about](https://github.com/hacken-in/website/blob/master/.travis.yml#L3) [this](https://github.com/hacken-in/website/blob/master/Gemfile#L3)?

**Comment by bitboxer** [comment]
http://gifs.joelglovier.com/eddie-izzard/not-listening.gif

**Comment by moonglum** [comment]
:dancers:

**Comment by bascht** [comment]
## Alrighty. Paging @moonglum & @bitboxer.
Deduplicated that second occurance of the Unicorn port with an `ERB Template`, so now it's your turn to render all our Ruby versions from one single source of truth. :trollface:

**Comment by moonglum** [comment]
Delightful, you may :shipit: Thanks for being such a glorious role model for DRYnes :heart:

**Pull request merged**